bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx82.html

```diff
@@ -0,0 +1,107 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import send_to_device with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block html_attrs %}
+data-whatsnew80-redirect-url="{{ settings.FXA_ENDPOINT + 'sms/?' + utm_params }}"
+{% endblock %}
+
+{% block page_title %}{{ ftl('whatsnew80-page-title') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}
+  {{ ftl('whatsnew80-page-title') }}
+{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_82') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-md">
+        <img class="wnp-main-image" src="{{ static('img/firefox/whatsnew/whatsnew80/hero-fastsync.svg') }}" width="200" alt="">
+        <h1 class="wnp-main-title">{{ ftl('whatsnew80-main-title') }}</h1>
+        <p class="wnp-main-tagline">{{ ftl('whatsnew80-main-intro') }}</p>
+
+      {% if show_send_to_device %}
+        <div id="send-to-device-wrapper" class="primary-cta">
+        {{ send_to_device(include_title=False, message_set='fx-whatsnew', input_label_alt=ftl('whatsnew80-primary-cta-description'), class='vertical') }}
+        </div>
+      {% else %}
+        <div class="qr-code-wrapper">
+          <img src="{{ static('img/firefox/mobile/protocol/qr-firefox.png') }}"
+              width="220"
+              height="220"
+              id="firefox-qr"
+              data-mozillaonline-link="{{ static('img/firefox/mobile/protocol/qr-firefox-mozillaonline.png') }}"
+              alt="">
+        </div>
+      {% endif %}
+
+    </div>
+  </section>
+
+  <section class="wnp-content-extra">
+    <div class="mzp-l-content mzp-t-content-xl">
+      <div class="l-columns-three">
```

Comment:
Note for reviewer: you can test the two-column layout locally by changing this to `l-columns-two` and removing one of the picto blocks.